packs/octopusdeploy/sensors/new_release_sensor.py

```diff
@@ -0,0 +1,135 @@
+import time
+
+from lib.sensors import OctopusDeploySensor
+
+
+class NewReleaseSensor(OctopusDeploySensor):
```

Comment:
This and the sensor above share a lot of common code. Ideally we could refactor common code into a base class and only override a couple of things which are needed in the sensor specific class (trigger_ref, _to_trigger, datastore key).